data/src/test/java/org/apache/iceberg/io/TestPositionDeltaWriters.java

```diff
@@ -79,13 +79,16 @@ public void setupTable() throws Exception {
   public void testPositionDeltaInsertOnly() throws IOException {
     FileWriterFactory<T> writerFactory = newWriterFactory(table.schema());
 
-    ClusteredDataWriter<T> dataWriter = new ClusteredDataWriter<>(
+    ClusteredDataWriter<T> insertWriter = new ClusteredDataWriter<>(
+        writerFactory, fileFactory, table.io(),
+        fileFormat, TARGET_FILE_SIZE);
```

Comment:
Does this need to wrap here?

Reply:
Wrapped for the consistency with the rest of the code around.